### ISSUE TITLE
Support for Kagi

### ISSUE THREAD

**Issue by xplosionmind** [cdeleeuwe/netlify-plugin-submit-sitemap#27]
It would be wonderful if the sitemap could be submitted to [Kagi](https://kagi.com), too.

**Comment by cdeleeuwe** [comment]
@xplosionmind do you know if Kagi offers a way to submit the sitemap? I could not find any documentation about it

**Comment by xplosionmind** [comment]
I do not… they are still in a very closed beta but the engine seems great. We could keep this issue open for future reference.

**Comment by cdeleeuwe** [comment]
Yes, sounds good. I will keep it open and label it as "blocked" for now.
